CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # LeanIX Kubernetes Connector Changelog
 
+## Release 2021-08-04 - 3.0.0
```

Comment:
since the update to defaults changes the behavior of the system I updated to 3.0.0. Also keeps the versioning simple there is no hard dependency on that